back-end/.env.dev

```
@@ -1,5 +1,7 @@
-DB_HOST=database
+DB_HOST=localhost
+DB_HOST2=database
 DB_PORT=5432
-DB_USERNAME=transland
+DB_USERNAME=postgres
+DB_USERNAME2=transland
```

```suggestion
DB_HOST=localhost
DB_PORT=5432
DB_USERNAME=postgres
```

`DB_HOST2`, `DB_USERNAME2` 는 안쓰이는 값 같아요.

---

일단은 저희가 도커 기준이 아니라 로컬 기준으로 연동을 해놔서 살짝 백업을 해둔 상태에요~